priv/gettext/de/LC_MESSAGES/default.po

```diff
@@ -411,6 +411,11 @@ msgstr ""
 msgid "Date"
 msgstr ""
 
+#, elixir-format
+#: lib/school_house_web/templates/layout/_lesson_menu.html.heex:312
+msgid "Debugging"
+msgstr ""
```

Comment:
I think this file needs an update here as well to `"on Twitter."`

https://github.com/elixirschool/school_house/blob/3bdcac9f5128329f80e6aac0475291e6e6c13a1c/priv/gettext/de/LC_MESSAGES/default.po#L936-L939

Reply:
Good catch! Fixed in https://github.com/elixirschool/school_house/pull/197/commits/9fb2dfd058cff81fb5738174192eb5aff22d61a5